server/lib/routes/get-index.js

```diff
@@ -2,6 +2,20 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+var activityEvent = require('../activity-event');
+var crypto = require('crypto');
+
+var activityEventQueryParams = [
+  'context',
+  'entrypoint',
```

Comment:
I realize a minority of users sign in/up via oauth flows, but is it worthwhile recording the `client_id` too?